File: Proyecto/src/main/java/javas/project/booleans/code1.java
package javas.project.booleans;

import org.apache.spark.api.java.JavaSparkContext;
import org.apache.spark.SparkConf;

public class code1 {
    public static void main(String[] args) {
        SparkConf conf = new SparkConf().setAppName("SparkDemo").setMaster("local");
        JavaSparkContext sc = new JavaSparkContext(conf);
        System.out.println("Spark version: " + sc.version());
        sc.stop();
    }
}

File: Proyecto/src/main/scala/scalas/project/booleans/code1.scala
package scalas.project.booleans


object code1 {
  def main(args: Array[String]) = {
    import org.apache.spark.sql.SparkSession
    val spark = SparkSession.builder.master("local[1]").appName("SparkByExample").getOrCreate

    val data = Seq((1, "John"), (2, "Jane"), (3, "Bob"))
    val df = spark.createDataFrame(data).toDF("id", "name")
    df.show()
    spark.stop()
  }
}
